CardanoSharp.Wallet/Extensions/Models/Transactions/TransactionWitnesses/VKeyWitnessExtensions.cs

```diff
@@ -69,5 +70,28 @@ public static VKeyWitness DeserializeVKeyWitness(this byte[] bytes)
         {
             return CBORObject.DecodeFromBytes(bytes).GetVKeyWitness();
         }
+
+        public static void CreateMocks(this ICollection<VKeyWitness> witnesses, int mocks)
```

Comment:
definitely more useful being able to just call it on the vkeywitness collection. nice